ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -148,6 +149,7 @@ void handleResponse(ResponseInfo responseInfo) {
     int correlationId = ((GetRequest) responseInfo.getRequest()).getCorrelationId();
     // Get the GetOperation that generated the request.
     GetRequestInfo getRequestInfo = correlationIdToGetRequestInfo.remove(correlationId);
+    routerMetrics.routerRequestLatencyMs.update(time.milliseconds() - getRequestInfo.startTimeMs);
```

Comment:
I am not very sure if `routerRequestLatencyMs` should be updated here. Are you measuring the time difference between start-to-process-response time and request-submitted time? It makes more sense to move this to the end of this method.